### DISCUSSION TITLE
4ndr0Glow

### DISCUSSION THREAD

**Discussion by 4ndr0666** [MarcinOrlowski/html-clock-plasmoid#63] [HTML Clock templates]
```html
<html>
<body>
<center>
<span style="font: ProggyCleanTT Nerd Font; font-size: 35px; font-weight: bold; color: #5beedc;">{k}</span>
<span style="font: ProggyCleanTT Nerd Font; font-size: 25px; color: #5beedc;">:</span>
<span style="font: ProggyCleanTT Nerd Font; font-size: 30px; color: white;">{ii}<span style="font-size: 20px; color: #cccccc;"><sup>{ss}</sup></span></span>
</center>
</body>
</html>
```

![2023-02-04_17-15](https://user-images.githubusercontent.com/97570105/216793595-6e5c19f0-838e-4f45-94bd-b90c09cc6c39.png)
![2023-02-04_17-17](https://user-images.githubusercontent.com/97570105/216793596-2047826c-fa6c-45ab-9040-6eeb1283027d.png)

** Notes **

* [x ] I confirm this is my original work.
* [x ] I attached screenshot of my layout to this ticket.
* [x ] I grant full and unrevocable rights to use the layout with HTML Clock widget free of charge.
* [x ] I grant users of HTML Clock widget right to use my layout free of charge and freely alter my layout if they need to.
* [x ] I want to be credited as <4ndr0666>.
* [x ] This is licensed under MIT

**Comment by MarcinOrlowski** [comment]
Thanks for your contribution.
